src/GHC.hs

```diff
@@ -199,7 +201,7 @@ stage1Packages = do
           ++ [ win32    | win     ]
 
 stage2Packages :: Action [Package]
-stage2Packages = return [haddock]
+stage2Packages = return [haddock, checkPpr, checkApiAnnotations]
```

Comment:
We don't want to build these two packages by default, so they shouldn't be added here.

Instead, `need` then in the `validate` rule.

Reply:
I think that intead, I can export packages as an [package] and make it as a need for  validate rule. It should do this for us.

Reply:
@chitrak7 Are you saying you'd like to do something like this?
```haskell
-- In GHC.hs
testsuitePackages :: [Package]
testsuitePackages = [checkPpr, checkApiAnnotations]

-- Rules/Test.hs
...
needPackages testsuitePackages
...
```
I think I like this. You'll need to define a helper function `needPackages` for that.

Reply:
Yes, exactly what i wish to do. Will implement this after recent pulls are merged

Reply:
Great, thank you!